src/scripts/main.js

```diff
@@ -1,3 +1,36 @@
 'use strict';
 
 // write code here
+document.addEventListener('DOMContentLoaded', () => {
+  const tableHeaders = document.querySelectorAll('th');
+  const tableBody = document.querySelector('tbody');
+
+  const tableRows = Array.from(tableBody.querySelectorAll('tr'));
```

Comment:
You're querying for table rows immediately after the DOM content is loaded. But these rows are not used until a header is clicked. It would be more efficient to move this line inside the click event handler.